Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -107,8 +107,14 @@
 
 void RemoteLayerTreeDrawingAreaProxy::remotePageProcessCrashed(WebCore::ProcessIdentifier processIdentifier)
 {
-    if (m_remoteLayerTreeHost)
+    if (!m_remoteLayerTreeHost)
+        return;
+
+    if (auto* scrollingCoordinator = m_webPageProxy->scrollingCoordinatorProxy()) {
+        scrollingCoordinator->willCommitLayerAndScrollingTrees();
         m_remoteLayerTreeHost->remotePageProcessCrashed(processIdentifier);
+        scrollingCoordinator->didCommitLayerAndScrollingTrees();
```

Comment:
Can we ever be in a situation where `scrollingCoordinator` is null, but `m_remoteLayerTreeHost` is non-null, and we still need to call `remotePageProcessCrashed`?

Reply:
This code reads oddly; the willCommitLayerAndScrollingTrees/didCommitLayerAndScrollingTrees pair aren't surrounding a real commit here, so we should rename them, or otherwise refactor.

Reply:
I think we can land as-is to restore the test and fix the crash on the bots and follow up with an improved naming or pattern as soon as possible.